locales/en.default.schema.json

```diff
@@ -11,9 +11,15 @@
           "label": "Accent 1",
           "info": "Used for solid button background."
         },
+        "gradient_accent_1": {
+          "label": "Accent 1 gradient"
+        },
```

Comment:
The look of it is a bit cluttered. Would be nice to some kinda hierarchy/grouping. But for future polish/improvement. Also it looks like the look of the settings has been updated 🤔 
[screenshot](https://screenshot.click/21-08-vlcqw-qxm8k.png)